mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
```

Comment:
The function name `eraseOrder` is not following the conventional naming style in JavaScript, which is camelCase. It should be renamed to `eraseOrder` -> `eraseorder`.